WindowsServerDocs/identity/ad-ds/active-directory-functional-levels.md

```diff
@@ -160,6 +160,7 @@ Supported Domain Controller Operating System:
 
 Supported Domain Controller Operating System:
 
+* Windows Server 2016 RS1
```

Comment:
```suggestion
* Windows Server 2016
```

Reply:
hm, I'm not sure we want to get into the RS1-RS5 release levels.  We would need to mention all SACs (ServerCore only). That would be:
- Windows Server 2016 LTSC
- Windows Server 2016 RS1
- Windows Server 2016 RS2
- Windows Server 2016 RS3
- Windows Server 2016 RS4
- Windows Server 2019 LTSC
- Windows Server 2019 1809
- Windows Server 2019 1903
- Windows Server 2019 1909
- Windows Server 2019 2004

Most on premise ADS customers don't use the SACs, and the few who do understand the general OS versions behind it. And they are not relevant in this context. the main message after all is to get away from legacy DCs and DFL/FFL levels.

Reply:
Yes, I think it's fine to avoid listing each individual release number since the broader concept is to move away from unsupported versions. That said, do lines 165-167 need to removed here, too?